src/jsonizer/internal/util.d

```diff
@@ -88,6 +94,18 @@ template findAttribute(alias attr, alias sym) {
   alias findAttribute = Filter!(match, __traits(getAttributes, sym));
 }
 
+/// Check is value is equal an initial of this type value
+bool isInitial(T)( T val )
+{
+  static if (is(typeof(val == T.init))) {
```

Comment:
It's need in case class comparison with null.
```d
class A{}
void foo() { assert( new A == A.init ); }
```
`clseq.d(2): Error: use 'is' instead of '==' when comparing with null`